plugin/src/main/java/org/opensearch/ml/sdkclient/RemoteClusterIndicesClient.java

```diff
@@ -71,6 +76,9 @@
 @Log4j2
 public class RemoteClusterIndicesClient implements SdkClientDelegate {
 
+    private static final String HASH_KEY = "_tenant_id";
```

Comment:
Query field should be `tenant_id`

Reply:
Query field should reference the CommonValue constant instead of copy/pasting the same String everywhere